packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -39,6 +39,14 @@ type BlockedMessageContentProps = {
   strings: MessageThreadStrings;
 };
 
+// Missing from html-to-react.
+type ProcessingInstructionType = {
```

Comment:
Curious why this was removed. But I think instead of having type in our `ChatMessageContent` it may fit better in its own file under the `acs-ui-common` package

Reply:
Nothing was removed :)
The html-to-react package was bumped to the latest version and they have released types. They just don't have _good_ types, and I think we should in the long term write our own version of this code.